lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -225,6 +225,21 @@ <h2>Do better web tester page</h2>
 <button class="small-button">Do something</button>
 <button class="small-button">Do something else</button>
 
+<!-- FAIL - frame-focusable-content -->
+<iframe tabindex="-1" src="focusable-iframe.html"></iframe>
+<!-- FAIL - nested-interactive -->
+<button><span tabindex="0">fail</span></button>
+<!-- FAIL - empty-table-header -->
+<table>
+  <tr>
+    <th></th>
+  </tr>
+</table>
+<!-- FAIL - aria-text -->
+<div role="text">
+  Still an <a href="#">interactive link</a> because of the author error.
+</div>
```

Comment:
move to `a11y_tester` with the other accessibility smoke tests?

Reply:
These are here to populate the sample LHR.

Reply:
> These are here to populate the sample LHR. Do we care about coverage there?

ah, got it. Ideally yes, so maybe just duplicate in there? It's very helpful to have `yarn smoke a11y` cover (almost?) everything in the accessibility category